### DISCUSSION TITLE
A question about ModSecurity deprecation

### DISCUSSION THREAD

**Discussion by dvershinin** [corazawaf/coraza#1090] [Q&A] [answered]
The README of coraza, among other things mentions:

> ... soon to be abandoned Trustwave ModSecurity Engine

However, it appears that libmodsecurity is maintained and there's no plan to deprecate. Could you let us know if we know something about its deprecation?

**Comment by M4tteoP** [comment, marked as answer]
Hi @dvershinin, these lines of the readme deserve some care and have to be updated. Modsecurity custodianship has been transferred to the OWASP in Jan 2024 and now both Coraza and Modsecurity (alongside the CRS) live under the same umbrella.
You can read more https://www.trustwave.com/en-us/resources/blogs/spiderlabs-blog/trustwave-transfers-modsecurity-custodianship-to-the-open-worldwide-application-security-project/ and https://coreruleset.org/20240115/welcome-the-newest-addition-to-the-owasp-family-modsecurity/

**Reply by M4tteoP** [reply]
Updated via https://github.com/corazawaf/coraza/pull/1115
